glue/viewers/scatter/qt/tests/test_python_export.py

```diff
@@ -260,25 +261,17 @@ def test_full_sphere_degrees(self, tmpdir):
         self.viewer.state.x_att = self.data.id['a']
         self.viewer.state.y_att = self.data.id['b']
         self.assert_same(tmpdir)
+        for flags in product([True, False], repeat=2):
+            self.viewer.state.flip_lonaxis, self.viewer.state.flip_lataxis = flags
+            self.assert_same(tmpdir)
+
+    def test_full_sphere_degrees(self, tmpdir):
+        self.viewer.state.angle_unit = 'degrees'
+        self._fullsphere_common(tmpdir)
 
     def test_full_sphere_radians(self, tmpdir):
         self.viewer.state.angle_unit = 'radians'
-        self.viewer.state.plot_mode = 'aitoff'
-        self.viewer.state.x_att = self.data.id['c']
-        self.viewer.state.y_att = self.data.id['d']
-        self.assert_same(tmpdir)
-        self.viewer.state.plot_mode = 'hammer'
-        self.viewer.state.x_att = self.data.id['e']
-        self.viewer.state.y_att = self.data.id['f']
-        self.assert_same(tmpdir)
-        self.viewer.state.plot_mode = 'lambert'
-        self.viewer.state.x_att = self.data.id['g']
-        self.viewer.state.y_att = self.data.id['h']
-        self.assert_same(tmpdir)
-        self.viewer.state.plot_mode = 'mollweide'
-        self.viewer.state.x_att = self.data.id['a']
-        self.viewer.state.y_att = self.data.id['b']
-        self.assert_same(tmpdir)
+        self._fullsphere_common(tmpdir)
```

Comment:
Why are the other projections not tested anymore? Maybe you could make `plot_mode` an input parameter to `_fullsphere_common`?

Reply:
Oops, removing the other projections was not intentional. Will add a parameter as you suggest and check each projection

Reply:
Looks like I didn't forget the other projections after all - `_fullsphere_common` checks each plot type

Reply:
Ah sorry, I only looked at the code as far as rendered in the diff!

Reply:
So did I...I forgot I'd covered all the projections until I opened up my editor 😆